scripts/db_connect.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# This script requires google's cloud_sql_proxy on your PATH. One way to set this up:
+# $ brew cask install google-cloud-sdk
+# $ gcloud components install cloud_sql_proxy
+# $ PATH=$PATH:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin/
```

Comment:
```suggestion
# $ PATH="$PATH:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin"
```

`PATH` should be quoted and paths should not include a terminating slash.

Reply:
IMO quoting `PATH` seems odd and not necessary. But it also seems harmless, so why not. I guess some user might have a path element with spaces in it.